charts/chartmuseum/values.yaml

```diff
@@ -73,7 +73,7 @@ env:
     # enable Prometheus metrics
     ENABLE_METRICS: false
     # disable all routes prefixed with /api
-    DISABLE_API: true
+    DISABLE_API: false
```

Comment:
If we change the default value of the charts package directly, updating the charts package from upstream in the future will require updating here as well.

IMO, would it make more sense to modify componentplan? 🤔️

Reply:
I modified the chart version and can reproduce the new chart package.
It is also possible to modify componentplan.yaml directly. @bjwswang

Reply:
Both look good to me.  We can set this `DISABLE_API: false` now to make sure user makes least changes on componentplan to use this chartmuseum